File: Cpp/2974_minimum_number_game.cpp
// https://leetcode.cn/problems/minimum-number-game/
        
class Solution {
public:
    vector<int> numberGame(vector<int>& nums) {
        vector<int> res(nums.size());
        sort(nums.begin(), nums.end());
        for (int i = 0; i < nums.size(); i+=2) {
            res[i] = nums[i+1];
            res[i+1] = nums[i];
        }
        return res;
    }
};

File: Python3/2974_minimum_number_game.py
# https://leetcode.cn/problems/minimum-number-game/

class Solution:
    def numberGame(self, nums):
        """
        Inputs:
            nums: list[int]
        Outputs:
            res: list[int]
        """
        nums.sort()
        res = [0 for _ in range(len(nums))]
        for i in range(0, len(nums), 2):
            res[i] = nums[i+1]
            res[i+1] = nums[i]
        return res